### PR TITLE
[devtools] add some sweet, sweet polish

### LINKED ISSUE #4040: fix numbered list in JS get started
https://developers.google.com/web/tools/chrome-devtools/javascript/#step_2_pause_the_code_with_a_breakpoint

### PR DESCRIPTION
**Pull request by kaycebasques**
cleans up the homepage and fixes #4040

### CODE REVIEW

File: src/content/en/tools/chrome-devtools/index.md [hunk 1]
```diff
@@ -19,94 +22,135 @@ Dogfood: Looking for the latest, [Chrome Canary](https://www.google.com/intl/en/
...
 
-### Profiles
-<img src="images/profiles-panel.png" alt="Profiles Panel" class="attempt-right">
-Use the Profiles panel if you need more information than the Timeline provide, for instance to track down memory leaks.
+### Memory panel (previously Profiles panel)
+
+<img src="images/panels/memory.png" alt="Profiles Panel" class="attempt-right">
+Profile memory usage and track down leaks.
 
+<b>Note</b>: In Chrome 57 the Profiles panel was renamed to the Memory panel.
```
Comment (petele) [inline src/content/en/tools/chrome-devtools/index.md]: Please use the standard `Note: something` syntax so you get the DevSite styled note stuff.

File: src/content/en/tools/chrome-devtools/index.md [hunk 2]
```diff
@@ -19,94 +22,135 @@ Dogfood: Looking for the latest, [Chrome Canary](https://www.google.com/intl/en/
...
 
-### Application
-<img src="images/application-panel.png" alt="Application Panel" class="attempt-right">
-Use the Resources panel to inspect all resources that are loaded, including IndexedDB or Web SQL databases, local and session storage, cookies, Application Cache, images, fonts, and stylesheets.
+### Application panel (previously Resources panel)
+
+<img src="images/panels/application.png" alt="Application Panel" class="attempt-right">
+Inspect all resources that are loaded, including IndexedDB or Web SQL databases, local and session storage, cookies, Application Cache, images, fonts, and stylesheets.
+
+<b>Note</b>: In Chrome 52 the Resources panel was renamed to the Application panel.
```
Comment (petele) [inline src/content/en/tools/chrome-devtools/index.md]: Please use the standard `Note: something` syntax so you get the DevSite styled note stuff.

File: src/content/en/tools/chrome-devtools/index.md [hunk 3]
```diff
@@ -19,94 +22,135 @@ Dogfood: Looking for the latest, [Chrome Canary](https://www.google.com/intl/en/
...
 
-### Timeline
-<img src="images/timeline-panel.png" alt="Timeline Panel" class="attempt-right">
-Use the Timeline to improve the run time performance of your page by recording and exploring the various events that happen during the lifecycle of a site.
+### Performance panel (previously Timeline panel)
+
+<img src="images/panels/performance.png" alt="Timeline Panel" class="attempt-right">
+Improve the runtime performance of your page by recording and exploring the various events that happen during the lifecycle of a site.
+
+<b>Note</b>: In Chrome 57 the Timeline panel was renamed to the Performance panel.
```
Comment (petele) [inline src/content/en/tools/chrome-devtools/index.md]: Please use the standard `Note: something` syntax so you get the DevSite styled note stuff.

**Comment by kaycebasques** [comment]
Oh ya, @petele, I forgot to tell you... look at how the note displays when you've got a floating image next to it:

screen shot 2017-01-24 at 9 58 32 am

**Comment by petele** [comment]
When that floating issues appears, you can use a `<div class="clear-fix"></div>` immediately after the note, and it'll fix that issue.

**Comment by kaycebasques** [comment]
Like this?

    Note: foo bar.

    <div class="clear-fix"></div>

Because I'm still seeing the issue on the local server.

**Comment by petele** [comment]
Try it before the Note then. It depends on where the floated image is.

**Comment by kaycebasques** [comment]
@petele it's laid out like this:

```
### Memory panel (previously Profiles panel)

<img src="images/panels/memory.png" alt="Profiles Panel" class="attempt-right">

Profile memory usage and track down leaks.

Note: In Chrome 57 the Profiles panel was renamed to the Memory panel.

* [Fix Memory Problems](/web/tools/chrome-devtools/memory-problems/)
* [JavaScript CPU Profiler](/web/tools/chrome-devtools/rendering-tools/js-execution)

<div style="clear:both;"></div>
```

I tried:

```
### Memory panel (previously Profiles panel)

<img src="images/panels/memory.png" alt="Profiles Panel" class="attempt-right">

<div class="clear-fix"></div>

Profile memory usage and track down leaks.

Note: In Chrome 57 the Profiles panel was renamed to the Memory panel.

* [Fix Memory Problems](/web/tools/chrome-devtools/memory-problems/)
* [JavaScript CPU Profiler](/web/tools/chrome-devtools/rendering-tools/js-execution)

<div style="clear:both;"></div>
```

And:

```
### Memory panel (previously Profiles panel)

<img src="images/panels/memory.png" alt="Profiles Panel" class="attempt-right">

Profile memory usage and track down leaks.

<div class="clear-fix"></div>

Note: In Chrome 57 the Profiles panel was renamed to the Memory panel.

* [Fix Memory Problems](/web/tools/chrome-devtools/memory-problems/)
* [JavaScript CPU Profiler](/web/tools/chrome-devtools/rendering-tools/js-execution)

<div style="clear:both;"></div>
```

And neither worked

**Comment by petele** [comment]
Hmm, just tried it myself, you're right, there's no easy way to get it to do what we want. 

Two options:
1) Move the note to the top, immediately under the heading & above the image.
2) Leave it as is. 

My preference is the first, but I won't block on #2.

**Comment by kaycebasques** [comment]
OK, thanks for checking @petele. I moved the notes to the top. That looks good enough.

**Pull request merged**